src/output-json-ftp.c

```diff
@@ -99,7 +99,7 @@ static json_t *JsonFTPLogCommand(Flow *f, FTPTransaction *tx)
         TAILQ_FOREACH(response, &tx->response_list, next) {
             /* handle multiple lines within the response, \r\n delimited */
             uint8_t *where = response->str;
-            uint16_t length = response->len;
+            uint16_t length = (uint16_t) strlen((const char *)response->str);
```

Comment:
So here we have len that is strlen()+1 to account for the nul, right? So wouldn't it be simpler/more performant to use `response->len - 1`? Or `response->len ? response->len - 1 : 0`?

Reply:
Agreed; removing the `strlen` when the value is easily calculated will be more performant.